index.html

```diff
@@ -37,34 +42,43 @@ <h3 class="getintouch">
             </h3>
           </div>
           <ul class="social-media-icon">
-            <li><a class="social-media" href="#"><img src="./images/twitter.png" alt="twitter"></a></li>
-            <li> <a class="social-media" href="#"><img src="./images/linkedin.png" alt="linkedin"></a> </li>
-            <li><a class="social-media" href="#"><img src="./images/medium.png" alt="medium"></a> </li>
-            <li><a class="social-media" href="#"><img src="./images/github.png" alt="github"></a></li>
-            <li><a class="social-media" href="#"><img src="./images/anglelist.png" alt="anglelist"></a></li>
+            <li><a class="social-media" href="#"><img src="./images/mobile/twitter.png" alt="twitter"></a></li>
+            <li> <a class="social-media" href="#"><img src="./images/mobile/linkedin.png" alt="linkedin"></a> </li>
+            <li><a class="social-media" href="#"><img src="./images/mobile/medium.png" alt="medium"></a> </li>
+            <li><a class="social-media" href="#"><img src="./images/mobile/github.png" alt="github"></a></li>
+            <li><a class="social-media" href="#"><img src="./images/mobile/anglelist.png" alt="anglelist"></a></li>
           </ul>
         </div>
+        <div>
 
       </section>
-      <section class="section">
+      <!-- <section class="section"> -->
         <section class="second-section">
           <div class="section-2">
-            <div class="second-sec-div">
-              <div>
-                <img class="proj-img" src="images/Snapshoot Portfolio.png" alt="picture">
+            <div class="container1">
+            <div class="second-sec-div1">
+              <div >
+                <img class="proj-img" src="images//mobile/Snapshoot Portfolio.png" alt="picture">
+                <img class="proj-img1" src="images/mobile/Snapshoot Portfolio@2x.png" alt="picture">
+                
               </div>
+              <div class="B">
               <div>
                 <h2 class="proj-title">Tonic</h2>
+                <h2 class="proj-title1">Tonic</h2>
```

Comment:
- [ ] The project title should have the font-size 40px according to the figma design, currently they are set to 32px, please fix this for all project titles :+1: